cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/SingleBrokerStats.java

```diff
@@ -58,6 +76,16 @@ public boolean isEstimated() {
     entry.put(HOST, _host);
     entry.put(BROKER, _id);
     entry.put(BROKER_STATE, _state);
+    if (_utilByDisk != null && !_utilByDisk.isEmpty()) {
```

Comment:
if `_utilByDisk` can be `null`, should we check this at other relevant places in this class?